notebooks/integrations/llama3/rag-elastic-llama3.ipynb

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
_Commented on notebook `notebooks/integrations/llama3/rag-elastic-llama3.ipynb` Cell 15 Line 1_
```markdown
The embeddings are stored in a dense vector field of dimension `4096`. The dimension size comes from the size of the embeddings generated from `Llama3`.
```
is that right? the embedding is 4096 dims? Could we improve this with picking a smaller dim model instead if so.

_Respond and view the context [here](https://gitnotebooks.com/elastic/elasticsearch-labs/pull/266)._

Reply:
AFAIK and checked, there didnt seem to be any option to change that.